pages/linux/qm-help.md

```diff
@@ -0,0 +1,12 @@
+# qm help
+
+> Get help about specified command.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Get help about specified command:
```

Comment:
```suggestion
- Display help:
```
Since we will be updating the page description. I would suggest updating this command description too.